### ISSUE TITLE
No bibliography output when exporting from LaTeX to html with biblatex

### ISSUE THREAD

**Issue by pprw** [jgm/pandoc#1943]
I'm using LaTeX with BibLaTeX and biber for the bibliography. **Pandoc seems to be not able to handle a biblatex bibliography when exporting to html**.

Here is a example with minimal .tex and .bib files:

*biblio.bib*
```
@Book{author00:_title,
  author =   {Author},
  title =    {Title},
  publisher =    {Publisher},
  year =     2000}
```
*mwe.tex*
```
\documentclass[12pt,twoside,a4paper]{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[english]{babel}

\usepackage[autostyle=true]{csquotes}
\usepackage[style=authoryear-comp,
     hyperref,
     backend=biber,
     date=year]{biblatex}
\addbibresource{biblio.bib}
\usepackage[colorlinks]{hyperref}

\begin{document}

\noindent test\autocite{author00:_title},\\
test\footcite{author00:_title},

\printbibliography
\end{document}
```
*command*
```
pandoc -s mwe.tex -o mwe.html --bibliography biblio.bib  --biblatex -t html5
```
*html output*

![capture d ecran - 15022015 - 12 38 39](https://cloud.githubusercontent.com/assets/5262986/6203120/9e418068-b50f-11e4-8784-b389072c6c09.png)

If I remove `--biblatex` in the command pandoc works fine:

![capture d ecran - 15022015 - 12 40 51](https://cloud.githubusercontent.com/assets/5262986/6203124/e4a4d78a-b50f-11e4-8b3d-c5703d25f018.png)

**To sum up, pandoc fails to export references with `--biblatex` option is enable.**

P-S : I'm using pandoc 1.12.4.2 on Debian unstable.

**Comment by jgm** [comment]
I think you're misunderstanding the use of the `--biblatex` option.
Here's what the manual says:

    --biblatex
              Use  biblatex for citations in LaTeX output.  This option is not
              for use with the pandoc-citeproc filter or with PDF output.   It
              is  intended  for use in producing a LaTeX file that can be pro-
              cessed with pdflatex and bibtex or biber.

This is not for use with HTML output.  If you are targeting HTML, just leave
the option off.  pandoc-citeproc and pandoc will automatically handle biblatex
citations and bibliographies.

**Comment by pprw** [comment]
Indeed, my mistake. No bug here!
